deployment/sklearn/sklearn-census-end-to-end.ipynb

```diff
@@ -35,8 +46,6 @@
     "import os\n",
     "import time\n",
     "\n",
-    "import six\n",
-    "\n",
```

Comment:
`six` is the bane of Python environments everywhere, and it's not explicitly used in this notebook anyway.